apps/aehyperchains/test/aehc_contract_tests.erl

```diff
@@ -0,0 +1,702 @@
+-module(aehc_contract_tests).
+
+
+-include("../../aecontract/src/aect_sophia.hrl").
+-include("../../aecontract/include/aecontract.hrl").
+-include("../../aecontract/test/include/aect_sophia_vsn.hrl").
+-include_lib("aecontract/include/hard_forks.hrl").
+-include_lib("eunit/include/eunit.hrl").
+
+
+-define(cid(__x__), {'@ct', __x__}).
+-define(hsh(__x__), {'#', __x__}).
+-define(sig(__x__), {'$sg', __x__}).
+-define(oid(__x__), {'@ok', __x__}).
+-define(qid(__x__), {'@oq', __x__}).
+
+-define(ALOT, (1000000000000000000000000000000000000000000000000 * aec_test_utils:min_gas_price())).
```

Comment:
This might be a bit confusing, from `aec_test_utils`:
```
min_gas_price() ->
    Protocol = aec_hard_forks:protocol_effective_at_height(1),
    max(aec_governance:minimum_gas_price(Protocol),
        aec_tx_pool:minimum_miner_gas_price()).
```
If the test is ran on a `roma` protocol (ex. `make eunit-roma`) the value of the gas price would be 1. This plays bad with the mocking of the protocol later on.

Not really a bug, but maybe a trap :D

Reply:
This amount is not important at all anyway. As long as the funds are sufficient and the fees are high enough everything should work